app/webpacker/components/RegistrationsV2/Registrations/RegistrationList.jsx

```diff
@@ -115,7 +117,6 @@ export default function RegistrationList({ competitionInfo }) {
 
   const registrationsWithPsychsheet = useMemo(() => {
     if (psychSheet !== undefined) {
-      setPsychSheetSortBy(psychSheet.sort_by);
```

Comment:
Why is this line being deleted?

Reply:
Otherwise, react throws an error that `setPsychSheetSortBy` is not defined

Reply:
Ah, because you're changing to a `useMemo` above. I see. But that means when loading a psych sheet from the backend, you cannot change the sort order anymore. I feel like this will break lots of existing use-cases

(for example, this will break proper sorting on events like 333 which have genuine use-cases of using either single _or_ average for sorting)